src/pages/Settings/SettingsPage.vue

```diff
@@ -87,6 +87,20 @@
 					</ul>
 				</a>
 			</section>
+			<section class="section">
+				<a
+					class="section__link"
+					href="/settings/payments"
+				>
+					<h2 class="section__title">Payment Settings</h2>
+					<ul class="section__list">
+						<li>
+							<h3>Payment methods</h3>
+							<p>Update your payment method details</p>
+						</li>
+					</ul>
+				</a>
```

Comment:
On second thought, this section reads fine... Maybe we just need to make the title of the new page "Payment Methods" or "Manage Payment Methods" to match this...